docker/docker-compose-local.yml

```diff
@@ -44,8 +44,8 @@ services:
   nginx:
     image: nginx:1.23.3
     volumes:
-      - ./local-nginx/gumroad_dev.conf:/etc/nginx/conf.d/default.conf
-      - ./local-nginx/certs:/etc/ssl/certs/
+      - ./local-nginx/gumroad_dev.conf:/etc/nginx/conf.d/default.conf:Z
```

Comment:
I'm not sure how well this plays out on OSX machine, but without ":Z" Nginx on SElinux (Fedora) doesn't work at all.